### DISCUSSION TITLE
How do you want to communicate outside GitHub?

### DISCUSSION THREAD

**Discussion by ayo-run** [ayoayco/astro-reactive-library#202] [Q&A] [answered]
Hi guys. I want to ask you if you want to have some more communication outside GitHub. It's important to me that you are comfortable about it so I don't want to just go ahead create a group chat somewhere. 

Couple of things I'm thinking
- Communication on GitHub is already working. We are getting some issues solved and I think that's easier to manage because this is something we do in our own time. So it is totally fine with me if we continue how we are. 
- But also maybe you want somewhere you can ask questions or send ideas in a more casual way and get more immediate feedback

I am okay with both. And I think it may even be easier for someone new to just ask stuff in a group chat rather than here. 

What do you think?

**Comment by ayo-run** [comment, marked as answer]
Tagging all awesome recent contributors :) @fazzaamiarso @Lalit3716 @Preet-Sojitra @NeilAn99 @Icyscools @alexsam29

**Reply by fazzaamiarso** [reply]
@ayoayco I think Discord is the standard for open-source projects. But, I'm fine with anything.

**Reply by Icyscools** [reply]
@ayoayco I think that is a great idea! It would be great if we have a chat room for small talk or to discuss some topics (perhaps fixing bugs). 🤣

**Reply by Preet-Sojitra** [reply]
> Tagging all awesome recent contributors :) @fazzaamiarso @Lalit3716 @Preet-Sojitra @NeilAn99 @Icyscools @alexsam29

If creating group outside Github, then discord is best option in my opinion else Github is fine for now

**Reply by Lalit3716** [reply]
> Tagging all awesome recent contributors :) @fazzaamiarso @Lalit3716 @Preet-Sojitra @NeilAn99 @Icyscools @alexsam29

I am fine with communicating through github but yeah if we want to create a group outside github then Discord is the  way to go!

**Reply by ayo-run** [reply]
I just got back from a frontend conference and excited to tell you about some excitement around our project! I also want to open the discussion about starting a github org, and who will be interested to have a maintainer role :)

Although, please checkout https://matrix.org as well since I'm hearing other bigger projects are regretting having their comms on discord and wish they have set up Matrix 😅

**Reply by Lalit3716** [reply]
> I just got back from a frontend conference and excited to tell you about some excitement around our project! I also want to open the discussion about starting a github org, and who will be interested to have a maintainer role :)
> 
> Although, please checkout https://matrix.org as well since I'm hearing other bigger projects are regretting having their comms on discord and wish they have set up Matrix 😅

Great to hear that project is gaining some attention! I am open to being a maintainer if you see me fit for it 😅.

By the way, Matrix seems complicated to set up and I am not sure if we should use it as I can't see any benefits of it honestly, but I am fine with either discord or matrix but slightly more with discord 😅.

**Reply by fazzaamiarso** [reply]
I agree with @Lalit3716. In my opinion, using Matrix can potentially become a barrier of entry for others as it's overshadowed by discord currently. 

Oh.. I'm also down to be a maintainer😁.

**Reply by ayo-run** [reply]
Thank you guys for the inputs and some interest getting more involved in the project! :) 

Looks like Discord is a popular choice. I'm going to look into opening a server, but will also do some thinking regarding pros & cons as I do it. Will update you how it goes!

**Reply by ayo-run** [reply]
Well that was easy. Join our Discord!!!

👉 https://discord.gg/4PRGW3QU

The only disadvantage I find is it may not be easy to move our data from Discord later when the community grows. But we'll just deal with that if/when we have to. 

See you there!
